Makefile

```diff
@@ -1,4 +1,7 @@
 UPTODATE := .uptodate
+SOURCE_VERSION := v0.11.0
+KUSTOMIZE_VERSION := v0.11.0
+HELM_CRD_VERSION := v0.9.0
 
 $(UPTODATE):
```

Comment:
Makefile etiquette: if you've defined a variable to stand in for a file, use that variable for every mention of that file. Or better, here you can just use `$@` in the rule body, to refer to the target. https://www.gnu.org/software/make/manual/html_node/Automatic-Variables.html

Reply:
I'll admit I cargo-culted this from other makefiles to try and fix a CI error that turned out to be  a mis-configured job. It can be removed.